be/src/formats/orc/apache-orc/c++/src/io/InputStream.hh

```diff
@@ -20,8 +20,7 @@
  * limitations under the License.
  */
 
-#ifndef ORC_INPUTSTREAM_HH
-#define ORC_INPUTSTREAM_HH
```

Comment:
Corresponding `#endif` need to be deleted

Reply:
Removed it!